packages/table/package.json

```diff
@@ -37,15 +37,17 @@
     "@tanstack/react-virtual": "^3.10.7",
     "lodash": "^4.17.21",
     "polished": "^4.2.2",
-    "react-fast-compare": "3.2.2"
+    "react-fast-compare": "3.2.2",
+    "react-intersection-observer":"^8.25.1"
   },
   "devDependencies": {
     "@faker-js/faker": "^8.0.0",
     "@leafygreen-ui/code": "^14.3.2",
     "@leafygreen-ui/button": "^21.2.0",
     "@leafygreen-ui/badge": "^8.1.2",
     "@leafygreen-ui/pagination": "^1.0.23",
-    "@lg-tools/storybook-utils": "^0.1.1"
+    "@lg-tools/storybook-utils": "^0.1.1",
+    "@emotion/styled": "^11.10.5"
```

Comment:
Does this break the live example on .design?

Reply:
it was, i think because we are importing the story file we need this to be a dependency, not a devDependencies. I've decided to remove this story and I will create a sandbox instead. I'll still test the TS for styled-components inside a spec file.